routes/messages/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+
+const router = express.Router();
+
+module.exports = (middlewares) => {
+  const { db } = middlewares;
+
+  router.get('/:id', async (req, res) => {
+    const id = req.query;
```

Comment:
I suggest changing this to `const { id } = req.params` as `req.params` should be used when you are trying to resolve route parameters.